apps/src/templates/sectionAssessments/SectionAssessments.jsx

```diff
@@ -106,12 +107,28 @@ class SectionAssessments extends Component {
     setScriptId(scriptId);
   };
 
-  showFreeResponseDetailDialog = questionText => {
-    this.setState({isDetailDialogOpen: true, detailQuestionText: questionText});
+  showFreeResponseDetailDialog = () => {
+    this.setState({
+      freeResponseDetailDialogOpen: true
+    });
   };
 
   hideFreeResponseDetailDialog = () => {
-    this.setState({isDetailDialogOpen: false, detailQuestionText: ''});
+    this.setState({
+      freeResponseDetailDialogOpen: false
+    });
+  };
+
+  showMulitpleChoiceDetailDialog = () => {
+    this.setState({
+      multipleChoiceDetailDialogOpen: true
+    });
+  };
+
+  hideMultipleDetailDialog = () => {
```

Comment:
Nit: Make this naming consistent with the `showMultipleChoiceDetailDialog`?

Reply:
Oops, definitely meant to do that.